envi/archs/h8/emu.py

```diff
@@ -0,0 +1,1314 @@
+"""
+The H8 Emulator module.
+"""
+
+import sys
+import struct
+
+import emu
+import envi
+import envi.bits as e_bits
+from envi.archs.h8 import H8Module
+from envi.archs.h8.regs import *
+from disasm import H8RegDirOper
+
+# calling conventions
+class H8ArchitectureProcedureCall(envi.CallingConvention):
+    """
```

Comment:
Is there a non-procedure or non-architecture call?  If not, i'd probably tighten this to H8Call.